tests/integration/test_authd/test_authd_key_hash.py

```diff
@@ -4,30 +4,45 @@
     Copyright (C) 2015-2021, Wazuh Inc.
     Created by Wazuh, Inc. <info@wazuh.com>.
     This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
-
-metadata:
-    component:
-        - Manager
-    modules:
-        - Authd
-    daemons:
-        - authd
-    operating_system:
-        - Ubuntu
-        - CentOS
-    tiers:
-        - 0
-    tags:
-        - Enrollment
-        - Authd
+type: integration
+tier:
+    0
+modules:
+    - Authd
+components:
+    - manager
+daemons:
+    - Authd
+path:
+    /tests/integration/test_authd/test_authd_key_hash.py
+os_platform
+    - linux
+os_version:
+    - Amazon Linux 1
+    - Amazon Linux 2
+    - Arch Linux
+    - CentOS 6
+    - CentOS 7
+    - CentOS 8
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 6
+    - Red Hat 7
+    - Red Hat 8
+    - Ubuntu Bionic
+    - Ubuntu Trusty
+    - Ubuntu Xenial
+tags:
+    - Enrollment
 '''
 
 import os
 import subprocess
 import time
 
 import pytest
-import yaml
 from wazuh_testing.tools import WAZUH_PATH
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.services import control_service
```

Comment:
Same as before about `read_yaml` on the line below and in `message_tests` (line 59)

Reply:
Fixed